app/preview/TemplateParams.scala

```diff
@@ -1313,72 +1313,6 @@ object TemplateParams {
       "commission" -> "2.55",
       "totalPaid" -> "126.00"
     ),
-    "ddi_cancelled" -> Map(
-      "FILE" -> "ADDACS",
-      "REASON" -> "1",
-      "NAME" -> "Customer",
-      "DDIREFERENCE" -> "123456789",
-      "REGIME" -> "VATC"
-    ),
-    "ddi_cancelled_online" -> Map(
-      "NAME" -> "Customer",
-      "DDIREFERENCE" -> "123456789"
-    ),
-    "ddi_not_acceptable" -> Map(
-      "FILE" -> "ADDACS",
-      "REASON" -> "B",
-      "NAME" -> "Customer",
-      "DDIREFERENCE" -> "123456789",
-      "REGIME" -> "VATC"
-    ),
-    "ddi_reinstated" -> Map(
-      "FILE" -> "ADDACS",
-      "REASON" -> "R",
-      "NAME" -> "Customer",
-      "DDIREFERENCE" -> "123456789",
-      "REGIME" -> "VATC",
-      "SORTCODE" -> "001122",
-      "ACCOUNTNUMBER" -> "12345678"
-    ),
-    "ddi_unpaid" -> Map(
-      "FILE" -> "ARUDD",
-      "REASON" -> "5",
-      "NAME" -> "Customer",
-      "DDIREFERENCE" -> "123456789",
-      "REGIME" -> "VATC",
-      "AMOUNT" -> "1000.00",
-      "COLLECTIONDATE" -> "20180501"
-    ),
-    "ddi_advance_notice" -> Map(
```

Comment:
May need to keep this one for Advance Notice For Hybrid Users. It's one that's sent directly to the email service rather than via DCS